src/UI/MvePlayer.cpp

```diff
@@ -908,9 +890,9 @@ void MvePlayer::think()
 {
     if (!_timerStarted) return;
 
-    struct timespec ts;
-    clock_gettime(CLOCK_MONOTONIC, &ts);
-    uint32_t nsec = (ts.tv_nsec - _lastts.tv_nsec);
+    struct TimeInfo ts;
+    CrossPlatform::getTime(&ts);
+    uint32_t nsec = (ts._nano - _lastts._nano);
```

Comment:
Type of the `TimeInfo::_nano` is `unsigned long long` which is std:: uint64_t I suppose.
And `nsec` variable should be the same.
Use `auto` keyword to keep it simple.